apps/condo/domains/billing/access/BillingIntegrationOrganizationContext.js

```diff
@@ -13,7 +13,7 @@ async function canReadBillingIntegrationOrganizationContexts ({ authentication:
     if (user.isAdmin || user.isSupport) return true
     return {
         OR: [
-            { organization: { employees_some: { user: { id: user.id }, role: { canManageIntegrations: true }, isBlocked: false } } },
+            { organization: { employees_some: { user: { id: user.id }, role: { OR: [{ canReadBillingReceipts: true }, { canManageIntegrations: true }] }, isBlocked: false } } },
```

Comment:
OR works like JOIN in graphql, please add test that:

`cant be read by organization employee without canReadBillingReceipts, but with other permissions`